Dockerfile

```diff
@@ -117,6 +117,7 @@ RUN mkdir -p /storage/uploads && \
 # Executables
 RUN chmod +x /usr/local/bin/doctor && \
     chmod +x /usr/local/bin/patch-create-missing-schedules && \
```

Comment:
can you now delete the old patch file ?

Reply:
Just bin file, or the while patch task?
We might want to keep it for historical reasons. Allows copy&pasting when the similar patch is needed. Feels more safer than relaying on Migration files

Reply:
hmm, okay you can keep the actual file but remove the binary

Reply:
Removed bin file